vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -47,6 +49,7 @@ fi
 # Use $SIG_IMAGE_VERSION for the sig resource, and $CAPTURED_SIG_VERSION (a random number) for the disk resource
 sig_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/${SIG_GALLERY_NAME}/images/${SIG_IMAGE_NAME}/versions/${SIG_IMAGE_VERSION}"
 disk_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/disks/${CAPTURED_SIG_VERSION}"
+capture_benchmark "set_variables"
```

Comment:
could we prefix the event names with the name of the file? e.g. `convert-sig-to-classic-storage-account/set_variables`?

Reply:
Would we be doing that for readability in the code or for the database schema? 

Currently each script is a key-value pair in the [JSON file](https://github.com/Azure/AgentBaker/blob/zb/regression2/vhdbuilder/packer/buildperformance/testdata/test-build-performance-data.json), with the key being the script name and the value being a map of string[string] (event name/timestamp). When the data is analyzed, the JSON collapses and each script name is appended to each event name that it contains. So when we're working with the data in Kusto, this would be `convert_sig_to_classic_storage_account_blob_set_variables`. 

Does that mitigate any concerns? I have a some regex expressions in the query for prefixes/suffixes so if possible it'd be nice to keep it this way.

Either way I can make this title a bit more expressive though, I understand it seems ambiguous by itself. @cameronmeissner

Reply:
Added them in